laurent22/joplin
# Desktop: Link "browse all plugins" to joplinapp.org/plugins

# Summary

Updates the "browse all plugins" link so that it opens the new plugin website, rather than the plugin repository GitHub page.

# Testing plan

1. Open the plugins tab in settings (on desktop)
2. Click on the gear icon
3. Click "browse all plugins"
4. Verify that the plugin website opens
